edge-hub/src/Microsoft.Azure.Devices.Routing.Core/IMessageStore.cs

```diff
@@ -3,6 +3,7 @@ namespace Microsoft.Azure.Devices.Routing.Core
 {
     using System;
     using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
```

Comment:
>using Microsoft.Azure.Devices.Edge.Util; [](start = 4, length = 40)

is this needed? The only new type I see in this file is uint

Reply:
Remnant. Removed.